pkg/component/kubernetes/apiserver/deployment.go

```diff
@@ -987,11 +964,6 @@ func (k *kubeAPIServer) vpnSeedPathControllerContainer() *corev1.Container {
 		TerminationMessagePolicy: corev1.TerminationMessageReadFile,
 	}
 
-	if k.values.VPN.DisableNewVPN {
```

Comment:
Let's just return the container struct directly and save a few more lines, i.e. `return &corev1.Container{...`.